examples/shop-page/default.project.json

```diff
@@ -0,0 +1,21 @@
+{
+	"name": "Roact Test Place",
```

Comment:
should we commit the default.project.json? Folks looking at this example may not know what Rojo is. Alternatively we could use add a README to this folder as a plug for Rojo for folks who want to quickly sync this into their game.

Reply:
I think I'm going to remove that! I just removed the client script to instead make the example loadable with the others from the `place.project.json` rojo configuration.